setup.py

```diff
@@ -49,9 +49,11 @@
     ],
     'postgresql': [
         'invenio-db[postgresql,versioning]>=1.0.0a9',
+        'psycopg2',
```

Comment:
The problem is not the compose, but the previous step, when extracting the deps from the setup.py.